packages/rum/src/boot/rum.spec.ts

```diff
@@ -114,6 +83,34 @@ describe('rum session', () => {
     expect(subsequentRequests[0].view.id).not.toEqual(initialRequests[0].view.id)
   })
 
+  it('when the session is renewed, a new view event should be sent', () => {
```

Comment:
Some of the remaining tests are integration tests, it could be interesting to see what we want to do with them at some point.